### ISSUE TITLE
checkcommits failing

### ISSUE THREAD

**Issue by jcvenegas** [01org/cc-oci-runtime#967]
checkcommits is failing with some PRs lets ignore if fails but keep it running to know what uses cases we are misssing.

**Comment by dlespiau** [comment]
Is this fixed then?

**Comment by jodh-intel** [comment]
Hi @dlespiau - for this repo which is only using TravisCI, yes. It fails under SemaphoreCI, but that appears to be due to a bug in the Semaphore environment (I've raised the issue with their support group).

@jcvenegas - I've reviewed all the PRs this week and `checkcommits` seems to be doing the right thing. If you agree, I'll raise a PR to remove the `||true` from https://github.com/01org/cc-oci-runtime/blob/master/.ci/ci-pre-checks.sh#L34 ?
